src/components/reports/ReportForm.tsx

```diff
@@ -64,7 +67,13 @@ const ReportForm: FC<Props> = (props: Props) => {
     ctx.client
       ?.request(bundle)
       .then((response) => console.debug("Bundle submitted", bundle, response))
-      .catch((error) => console.error(error));
+      .catch((error) => {
+        console.error(error)
+        setModal({
+          message: 'Something went wrong submitting the bundle. Please try again later.',
+          isError: true,
+      });
+      });
     actions.setSubmitting(false);
```

Comment:
probably also worth going through the response entries and if any of the response codes don't start with `2`, then add a modal message to saying that there was an error with the FHIR resource. If you'd like some example bundles which will cause an error let me know and I can make some for you